recipes/daggy/2.1/conanfile.py

```diff
@@ -0,0 +1,133 @@
+from conans import ConanFile, CMake, tools, errors
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.43.0"
+
+
+class DaggyConan(ConanFile):
+    name = "daggy"
+    license = "MIT"
+    homepage = "https://daggy.dev"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Data Aggregation Utility and C/C++ developer library for data streams catching"
+    topics = ("streaming", "qt", "monitoring", "process", "stream-processing", "extensible", "serverless-framework", "aggregation", "ssh2", "crossplatform", "ssh-client")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "with_ssh2": [True, False],
+        "with_yaml": [True, False],
+        "with_console": [True, False],
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "with_ssh2": True,
+        "with_yaml": True,
+        "with_console": False,
+        "shared": False,
+        "fPIC": False
```

Comment:
```suggestion
        "fPIC": True
```

By default, it should be true.

Reply:
Got an error - [HOOK - conan-center.py] pre_build(): ERROR: [FPIC MANAGEMENT (KB-H007)] 'fPIC' option not managed correctly. Please remove it for shared option: del self.options.fPIC (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H007)